builds/runtimes/pypy-5.1.0

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
The code in this file seems to repeat itself in several other files with only the parameters changing.